### DISCUSSION TITLE
Query regarding Creating json files

### DISCUSSION THREAD

**Discussion by aishwaryo** [FitSNAP/FitSNAP#79] [Q&A] [answered]
Hi all, I have query regarding the functioning of FITSNAP.To my understanding, we need to have the reference structure files as json format. Is it the only allowed format? How can I convert an OUTCAR file derived from an AIMD run using VASP into their contributing json files( the OUTCAR file has different structures belonging to the MD trajectory)? Thanks in advance.

**Comment by charlessievers** [comment]
@mind-dime We have implemented two formats for now, JSON and extXYZ. I believe some of the other authors have scripts to change VASP output to JSON that they may be willing to share. @athomps, do you have any suggestions for @mind-dime?
Best,
Charlie

**Comment by mitwood** [comment]
@mind-dime Thanks for your post. We are actively working on a VASP OUTCAR to JSON format converter and it should be available in the coming weeks. I will check with the author or this code and update this discussion.

**Reply by aishwaryo** [reply]
Thanks for your answers. I think the converter will be useful to the users of this software.

**Comment by mitwood** [comment, marked as answer]
@mind-dime We have added a VASP to JSON converter in the tools folder, please have a look. Thanks @ldwillia for the contribution.
